### DISCUSSION TITLE
Profile

### DISCUSSION THREAD

**Discussion by violethaze74** [community/community#36044] [Other Feature Feedback, Questions, & Ideas]
How do I merge a personal profile with an organizational profile?

**Comment by Anmol-Baranwal** [comment]
You want to create an organization profile right? In that case, the below instructions might help:

Warning: Before converting a user into an organization, keep these points in mind.

- You will no longer be able to sign into the converted personal account.
- You will no longer be able to create or modify gists owned by the converted personal account.
- An organization cannot be converted back to a user.
- The SSH keys, OAuth tokens, job profile, reactions, and associated user information, will not be transferred to the organization. This is only true for the personal account that's being converted, not any of the personal account's collaborators.
- Any commits made with the converted personal account will no longer be linked to that account. The commits themselves will remain intact.
- Any existing comments made by the converted personal account will no longer be linked to that account. The comments themselves will remain intact but will be associated with the `ghost` user.
- Any forks of private repositories made with the converted personal account will be deleted.

> Create a new personal account, which you'll use to sign into GitHub and access the organization and your repositories after you convert.
>[Leave any organizations](https://docs.github.com/en/articles/removing-yourself-from-an-organization) the personal account you're converting has joined.

Steps:
1. In the upper-right corner of any page, click your profile photo, then click Settings.

3. In the "Access" section of the sidebar, click  Organizations.
4. Under "Transform account", click Turn into an organization.
5. In the Account Transformation Warning dialog box, review and confirm the conversion.
6. On the "Transform your user into an organization" page, under "Choose an organization owner", choose either the secondary personal account you created in the previous section or another user you trust to manage the organization.
7. Choose your new organization's subscription and enter your billing information if prompted.
8. Click Create Organization.

**Reply by Anmol-Baranwal** [reply]
John-clark, do tell me the reason for which you have downvoted the answer so that I can improve my answer next time. 
If you don't provide the reason, then you are just downgrading the answer without regards for the community.

**Comment by backtoaldo** [comment]
In the github documentation that you can check [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization), you can follow the steps to convert your user to an organization. If this solution is not good, you can also transfer repositories to a new organization in a simple way: [documentation for repository transfer](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository). 
Hope you get the best solution for your problem🥂
